### DISCUSSION TITLE
Unable to find SpotBugs method

### DISCUSSION THREAD

**Discussion by toledobh** [spotbugs/spotbugs-gradle-plugin#730] [Q&A]
Hey, I just wanted to know if anyone has encountered the following error.
`Unable to find method 'com.github.spotbugs.snom.internal.SpotBugsTaskFactory.generate(Lorg/gradle/api/Project;Lorg/gradle/api/Action;)V'`
When using JDK 17, Gradle 7.4.2 and Spotbugs gradle plugin 5.0.6.

**Comment by KengoTODA** [comment]
This method was removed from v4.7.4 (8981c0e929fe5027e1f4ea0d4261be3af75b5933). So I guess that your classpath contains not only spotbugs-gradle-plugin 5.0.6 but also an older version.

Make sure you have only one spotbugs-gradle-plugin in your classpath.

**Reply by toledobh** [reply]
Thank you so much Kengo! I will do that.
